src/index.html

```diff
@@ -4,13 +4,49 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width,
+        user-scalable=no, initial-scale=1.0,
+        maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+      class="BigSearch">
+
+      <img
```

Comment:
https://mateacademyen.slack.com/archives/C03S1JURNG5/p1660665961358809
```suggestion
    <form
      action="#"
      method="get"
      data-qa="big"
      class="BigSearch"
    >
      <img
```